src/main/java/org/extism/chicory/sdk/ManifestModuleMapper.java

```diff
@@ -0,0 +1,53 @@
+package org.extism.chicory.sdk;
+
+import com.dylibso.chicory.aot.AotMachine;
+import com.dylibso.chicory.runtime.Module;
+
+class ManifestModuleMapper {
+    private final Manifest manifest;
+
+    ManifestModuleMapper(Manifest manifest) {
+        this.manifest = manifest;
+    }
+
+    Module.Builder toModuleBuilder() {
+        if (manifest.wasms.length > 1) {
+            throw new UnsupportedOperationException(
+                    "Manifests of multiple wasm files are not supported yet!");
+        }
```

Comment:
As stated above, we can do a follow up issue for this.

Reply:
yup, just making this more explicit for now